air/tests/test_module/negative_tests/uncatchable_trace_related.rs

```diff
@@ -576,13 +576,102 @@ fn fold_pos_overflow() {
             subtrace_len: wrong_after_subtrace_len,
         },
     );
-    let resolved_fold = ResolvedFold::new(
-        hashmap![value_pos.into() => fold_lore],
-        (wrong_after_subtrace_len + 1) as usize,
-    );
+    let resolved_fold = ResolvedFold::new(hashmap![value_pos.into() => fold_lore], wrong_after_subtrace_len + 1);
```

Comment:
I think the hash map should be formatted as
```
        hashmap!(
            value_pos.into() => fold_lore,
        ),
```
(Note the `( ... )`).

Reply:
According with the [maplit docs](https://docs.rs/maplit/1.0.2/maplit/) it doesn't matter which parenthesis to use.

Reply:
Fixed though.

Reply:
It does matter for style.

Reply:
From my experience the styling holywars are painful and demoralizing so this should be somehow coded into automated styling enforces.